src/engine/QueryPlanner.h

```diff
@@ -724,7 +724,8 @@ class QueryPlanner {
   void checkCancellation(ad_utility::source_location location =
                              ad_utility::source_location::current()) const;
 
-  // Return a filter that filters the active graphs.
+  // Return a filter that filters the active graphs. 'Active' graphs are graphs
+  // that are either implicitly or explicitly requested by SPARQL query.
```

Comment:
Can you be more specific, in particular:

1. Outside of `GRAPH` clauses, the default graphs (implicit, or specified via `FROM` ) are eactive, and inside a `GRAPH` clause, this speicifed via an IRI in the graph clause or via `FROM NAMED`.